### ISSUE TITLE
[Bug Report][3.0.1] V-Navigation-Drawer - Temporary V-Navigation-Drawer does not have focus trap

### ISSUE THREAD

**Issue by rafaeldsousa** [vuetifyjs/vuetify#16140]
### Environment
**Vuetify Version:** 3.0.1
**Vue Version:** 3.2.45
**Browsers:** Chrome 107.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Just open the drawer and note that the focus is not trapped in the overlay element. I can see the same behaviour happening for the v-menu, the overlay is created but not focus trap is added.

The docs for v-menu state that (no reference of this for v-navigation-drawer)
By default, v-menu components are detached and moved to the root of your application. In order to properly support inserting dynamic content into the DOM, you must use the attach prop. This will ensure that focus transfers from the activator to the content when pressing the tab key.

### Expected Behavior
Once drawer is open, focus should be trapped within new element. As per Material UI drawers
https://mui.com/material-ui/react-drawer/

### Actual Behavior
Drawer and V-Menu is opened and focus is not set to overlay items.

### Reproduction Link
[https://codepen.io/rafaeldesousa/pen/abKqLgL](https://codepen.io/rafaeldesousa/pen/abKqLgL)

**Comment by atilkan** [comment]
Have the same problem.

**Comment by rafaeldsousa** [comment]
Seems like the problem is with the v-overlay component that does not set the focus when opened. 
Both v2-3 have the same behaviour. Open the overlay and hit tab, you'll see that the focus isn't within the recently opened overlay.

https://vuetifyjs.com/en/components/overlays/
https://next.vuetifyjs.com/en/components/overlays/

**Comment by jalamanderman** [comment]
I'm also experiencing this issue.

**Comment by atilkan** [comment]
Lets wait for this [PR](https://github.com/vuetifyjs/vuetify/pull/15934) to be merged.

**Comment by rafaeldsousa** [comment]
@atilkan I'm not entirely sure this relates to the focus being set on overlays or any dynamically created elements.
Looking at the [code for v-navigation-drawer](https://github.com/vuetifyjs/vuetify/blob/b0140b073ef1502d957d6234a29a0d274109c2b9/packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.ts#L385) doesn't look like the focus is set anywhere.

**Comment by rogeriobayer** [comment]
Any workarounds or fixes for this case? It will be fixed in 3.9.0?
